robocopkaka/bootcamp-29-project
# #158055951 Fix issues component tests

#### What does this PR do?
fix issues with component tests
#### Description of Task to be completed?
* fix errors when running frontend tests
* add more tests for components
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run `npm install` to install other dependencies
* run `npm run test:jest` to run frontend tests
* coverage should have increased and no errors
#### Any background context you want to provide?
tests were failing after using materialize components
#### What are the relevant pivotal tracker stories?(if applicable)
#158055951
